frontend/templates/product/sections/FeaturedProductsSection/index.tsx

```diff
@@ -108,13 +116,43 @@ function ProductGridItem({
    isPro,
    warranty,
    oemPartnership,
+   sku,
 }: ProductGridItemProps) {
    const discountPercentage = computeDiscountPercentage(price, compareAtPrice);
    const hasLifetimeWarranty =
       typeof warranty === 'string' && isLifetimeWarranty(warranty);
 
    const productHeadingId = `product-heading-${handle}`;
 
+   const appContext = useAppContext();
+   const fetchData = async () => {
+      if (!sku) {
+         return null;
+      }
+      const reviewsData = await fetchProductReviews(
+         appContext.ifixitOrigin,
+         sku
+      );
+      return reviewsData;
+   };
+
+   const [reviewsData, initReviews] = useState<
```

Comment:
Generally speaking, I think it's [convention](https://reactjs.org/docs/hooks-state.html#declaring-a-state-variable) to name the state update function like `setNameOfStateVar`. I think it makes it a little clearer later on that you're updating React state, as opposed to invoking some other kind of callback.

So here, I would personally name this
```suggestion
   const [reviewsData, setReviewsData] = useState<
```